vllm/v1/worker/gpu_model_runner.py

```diff
@@ -2786,6 +2796,32 @@ def load_model(self, eep_scale_up: bool = False) -> None:
                 self.model = UBatchWrapper(self.model, self.vllm_config,
                                            CUDAGraphMode.NONE, self.device)
 
+    def _get_eagle3_aux_layers_from_config(self) -> Optional[tuple[int, ...]]:
+        """Extract Eagle3 auxiliary layer IDs from speculative config.
```

Comment:
should be index values? not ids?
Can we say "use the list of indices to fetch the appropriate hidden states from the base model" or something more desciptive?